### ISSUE TITLE
Warning, new version of rvm available '1.29.6', you are using older version '1.29.7'.

### ISSUE THREAD

**Issue by ejstembler** [rvm/rvm#4535]
## Description

If you update to master, there's a waning complaining that there's a newer version available.  e.g.

`Warning, new version of rvm available '1.29.6', you are using older version '1.29.7'.`

## Steps to reproduce

Install the master brach one of two ways:

```shell
rvm get master
```
or
```shell
\curl -sSL https://get.rvm.io | bash -s master
```
Then try installing a new ruby:
```shell
rvm install 2.5.3
Warning, new version of rvm available '1.29.6', you are using older version '1.29.7'.
```

## Expected behavior

It's expected to know that 1.29.6 is less than 1.29.7

## Environment info

system:

  system:
    uname:        "Darwin Mac-mini.home 18.2.0 Darwin Kernel Version 18.2.0: Mon Nov 12 20:24:46 PST 2018; root:xnu-4903.231.4~2/RELEASE_X86_64 x86_64"
    name:         "OSX"
    version:      "10.14"
    architecture: "x86_64"
    bash:         "/bin/bash => GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)"
    zsh:          "/bin/zsh => zsh 5.3 (x86_64-apple-darwin18.0)"
    remote_path:  "osx/10.14/x86_64"
    xcode:        "10.1"

  rvm:
    version:      "1.29.7 (master)"
    updated:      "4 minutes 48 seconds ago"
    path:         "/Users/ejstembler/.rvm"
    autolibs:     "[4] Allow RVM to use package manager if found, install missing dependencies, install package manager (only OS X)."

  homes:
    gem:          "not set"
    ruby:         "not set"

  binaries:
    ruby:         "/usr/local/bin/ruby"
    irb:          "/usr/local/bin/irb"
    gem:          "/usr/local/bin/gem"
    rake:         "/usr/local/bin/rake"

  environment:
    PATH:         "/usr/local/opt/ruby/bin:/Users/ejstembler/.kiex/bin:/Users/ejstembler/bin:/Users/ejstembler/go/bin:/mongodb/bin:/Users/ejstembler/miniconda3/bin:/usr/local/opt/erlang@17/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/local/go/bin:/usr/local/share/dotnet:/opt/X11/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Applications/Postgres.app/Contents/Versions/latest/bin:/Users/ejstembler/bin/FDK/Tools/osx:/Users/ejstembler/.rvm/bin"
    GEM_HOME:     ""
    GEM_PATH:     ""
    MY_RUBY_HOME: ""
    IRBRC:        ""
    RUBYOPT:      ""
    gemset:       ""

**Comment by hivre** [comment]
I can confirm on a macbook pro (2017 model)

**Comment by kugaevsky** [comment]
Almost the same. RVM tries to update itself on every `rvm install x.x.x`.
```
➜  ~  rvm version
rvm 1.29.6 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]

➜  ~ rvm install 2.5.3
Found old RVM 1.29.6 - updating.
Downloading https://get.rvm.io
...
```
macOS Mojave 10.14.2

**Comment by KubaSemczuk** [comment]
I think the problem still persist in a little bit different example:
"Warning, new version of rvm available '1.29.7', you are using older version '1.29.7-next'."

'1.29.7-next' is current master, so it's newer version than stable '1.29.7', right?

**Comment by pelargir** [comment]
Yes, this is still happening:

`Warning, new version of rvm available '1.29.9', you are using older version '1.29.9-next'.`

**Comment by doc75** [comment]
> Yes, this is still happening:
> 
> ```
> Warning, new version of rvm available '1.29.9', you are using older version '1.29.9-next'.
> You can disable this warning with:   echo rvm_autoupdate_flag=0 >> ~/.rvmrc
> You can enable auto-update with:     echo rvm_autoupdate_flag=2 >> ~/.rvmrc
> You can update manually with:        rvm get VERSION                         (e.g. 'rvm get stable')
> ```

I confirm
